sklearn/decomposition/dict_learning.py

```diff
@@ -1036,18 +1108,23 @@ class MiniBatchDictionaryLearning(BaseEstimator, SparseCodingMixin):
     alpha : float,
         sparsity controlling parameter
 
+    l1_ratio: float,
+        sparsity controlling parameter, for 'elastic_net' method only
+
     n_iter : int,
         total number of iterations to perform
 
-    fit_algorithm : {'lars', 'cd'}
+    fit_algorithm : {'lars', 'cd', 'elastic_net'}
```

Comment:
I think this should be called `projected_gradient`. elastic_net by itself is not a solver like lars and coordinate descent but rather a kind of penalty.